pkg/system/phase2_creating.go

```diff
@@ -459,16 +459,23 @@ func (r *Reconciler) SetDesiredCoreApp() error {
 	podSpec.ServiceAccountName = "noobaa"
 	coreImageChanged := false
 
+	// adding the missing Volumes from default podSpec
+	podSpec.Volumes = r.DefaultCoreApp.Volumes
+
 	for i := range podSpec.Containers {
 		c := &podSpec.Containers[i]
+
+		// adding the missing VolumeMounts from default container
+		c.VolumeMounts = r.DefaultCoreApp.Containers[i].VolumeMounts
+		// adding the missing Env variable from default container
+		util.MergeEnvArrays(&c.Env, &r.DefaultCoreApp.Containers[i].Env)
```

Comment:
Can you also move the 2nd line up?
```suggestion
		util.MergeEnvArrays(&c.Env, &r.DefaultCoreApp.Containers[i].Env)
		r.setDesiredCoreEnv(c)
```

Reply:
Done, Thanks.